src/server.ts

```diff
@@ -119,10 +119,10 @@ class REMSServer extends Server {
    */
```

Comment:
I know this isn't within the scope of this PR, but the parameter types listed in `listen`'s docstring don't quite align with the actual types of `listen`'s parameters.

Reply:
it looks like the params `port` and `callback` are called in and used in this function in the return function

Reply:
Sorry, I should've been more clear in my previous comment.
```suggestion
  listen({ port }: { port: number }, callback: () => void) {
```

Although, I just saw this old GitHub issue from @projecttacoma/node-fhir-server-core that they [aren't planning on supporting TypeScript anytime soon for R4](https://github.com/bluehalo/node-fhir-server-core/issues/186). :(